### DISCUSSION TITLE
Hourly Activity

### DISCUSSION THREAD

**Discussion by npcwoody** [jlai/fitness-dashboard#33] [General]
Have you made any progress on "Hourly Activity"? I have mine setup to track 10 hours and alert me ten minutes before the hour if I  haven't walked at least 250 steps in the hour. It is one of the major things I use to track my progress for the day. As a person with heart issues it is very important to me. It helps me avoid unwanted naps during the day.

**Comment by jlai** [comment]
Sorry, I didn't get around to it earlier. I've added a new tile (Edit Grid -> Add Tile -> Hourly Step Goal). Click on it to configure the hours. It doesn't sync with or otherwise affect the reminders on your watch/tracker.

**Reply by npcwoody** [reply]
Thank you very much! Almost perfect for me! Have you considered a tile showing last reported weight and %fat? I know you have the trend graph, but a tile would be nice so I can tell at a glance.....

**Reply by jlai** [reply]
I've added a weight tile (`Edit Grid` -> `Add Tile` -> `Weight`) that shows the last logged weight -- only logged weights, not the trendline which is extrapolated. Right now it *just* shows the weight.

You can also click on it to compare to 30 days ago (or the closest logged weight) or another date within the last 30 days. (The 30 days is because that's the max days we can request from the Fitbit API at a time for weight logs; we could make multiple requests but that gets more complicated.)

**Reply by npcwoody** [reply]
Thanks again!!

**Reply by jlai** [reply]
Whoops had a bug with kilogram -> pounds conversion. Reload if you see a weird number on the weight tile.

**Reply by npcwoody** [reply]
Thats better! It said I was 709 pounds.... Now it is correct.

**Reply by npcwoody** [reply]
I know I am being a pest, but it would be nice if the dots on the hourly steps tile turned green when you achieve your goal like the other tiles do, If you can't, that's fine. at least the tile is there!!

**Reply by jlai** [reply]
For now I've made it so the icon in the middle turns green when you hit your daily target (dots will remain red)

**Reply by npcwoody** [reply]
Thanks, but I keep my tiles at their smallest size (so I have a bunch of them without scrolling) so I don't see the Icon in the middle. I can live with the red dots.

**Comment by npcwoody** [comment]
Thank you!!


⁣Woody Woodward
Sent from my phone 

Get BlueMail for Android ​

**Comment by chrissard2** [comment]
Thank you for the ability to doc Steps. Even the fitbit app does not, you have to convert your steps into miles to doc your activity.
